vnext/PropertySheets/OutputMSBuildProperties.targets

```diff
@@ -18,7 +18,8 @@
         "WindowsTargetPlatformVersion": "$(WindowsTargetPlatformVersion)",
         "UseExperimentalNuGet": "$(UseExperimentalNuGet)",
         "UseHermes": "$(UseHermes)",
-        "UseWinUI3": "$(UseWinUI3)"
+        "UseWinUI3": "$(UseWinUI3)",
```

Comment:
Not that I particularly like it, but you *could* include `UseFabric` in here just in case to catch edge cases.

Reply:
What edge cases would be covered by including `UseFabric` that `RnwNewArch` would not?